find_metrics.py

```diff
@@ -158,14 +192,18 @@ def find_nearby_enhancer_densities(gene_data, overlaps):
     
 def gene_scoring(genes):
     
+    #Various attributes of each gene are scaled and normallised, before being
```

Comment:
Rename to _assign_gene_score()_, or something. Remember your function names have got to be as seamless as possible to read from the place where they will be called